scripts/prepare_test_env.sh

```diff
@@ -30,6 +30,14 @@ export REQUESTS_CA_BUNDLE=`pwd`/ca.pem
 export MANAGEMENT_CA_CERT_PATH=`pwd`/ca.pem
 export CURL_CA_BUNDLE=`pwd`/ca.pem
 
+
+if [ "$MGT_API_AUTHORIZATION" == "true" ]; then
+  USER_PASSWD=`kubectl get secret imm-openldap-customldif -o yaml|grep immconfig|awk '{ print $2 }'|base64 --decode|grep userpassword|awk '{ print $2 }'`
+  USER_NAME=user@example.com
+  export SINGLE_TENANT_USER=$USER_NAME:$USER_PASSWD
+  echo "Single tenant credentials : $SINGLE_TENANT_USER"
```

Comment:
Maybe there could be a nicer way to deliver the password to a user than printing it out? I believe it's not crucial, but still this solution is not the best method I guess

Reply:
I think this is quite ok to print password here, because user next step would be to log in to the platform. He can always change password later.